src/integrationTest/groovy/wooga/gradle/release/ReleasePluginIntegrationSpec.groovy

```diff
@@ -46,6 +46,26 @@ class ReleasePluginIntegrationSpec extends IntegrationSpec {
         createFile("paket.dependencies")
     }
 
+    def "verify paket unity install is called before any unity task is executed"() {
```

Comment:
nice! 

did you happend to TDD-ish this? e.g. did you run this test on the negative case, when this fix wasn't applied yet ?

Reply:
Well locally yes. But I don't provide a negative test case here because the feature is not configuratble.

Reply:
To make this clearer and saver I should actually call multiple other Unity tasks and define custom ones. To actually proof this.

Reply:
I take it as is for now though.